src/System.Private.CoreLib/shared/Internal/Resources/WindowsRuntimeResourceManagerBase.cs

```diff
@@ -2,32 +2,37 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Globalization;
 
 namespace Internal.Resources
 {
-    // This is implemented in System.Runtime.WindowsRuntime as System.Resources.WindowsRuntimeResourceManager,
-    // allowing us to ask for a WinRT-specific ResourceManager.
-    public abstract class WindowsRuntimeResourceManagerBase
+    // This implementation wraps the WindowsRuntimeResourceManager implementation in ProjectN to adapt it to the new IWindowsRuntimeResourceManager interface
```

Comment:
We are not longer syncing CoreLib with ProjectN. You do not need to worry about keeping ProjectN working.